config.json

```
@@ -361,6 +361,14 @@
         "prerequisites": [],
         "difficulty": 4
       },
+      {
+        "slug": "armstrong-numbers",
+        "name": "Armstrong Numbers",
+        "uuid": "3dd0d34b-1584-4960-bf82-beb2f4ca4e48",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 1
```

```suggestion
        "difficulty": 2
```
suggestion: i think the difficulty is at least 2. 1 is more like hello world-level exercises. I'd probably put it at 3, but 2 splits the difference between us. What do you think?

---

You're right, it's not a '1'. I'm comfortable with level '3'. Will make changes.